platform/ios/docs/guides/Adding Points to a Map.md

```diff
@@ -0,0 +1,80 @@
+# Adding points to a map
+
+Mapbox offers a few different ways to add points to a map, each with different tradeoffs.
+
+##Adding annotations to a map
+
+It's straightforward to add an annotation to a map. You can use `MGLPointAnnotation` as is, or you can subclass it to add annotations with richer data.
+
+```swift
+let annotation = MGLPointAnnotation()
+annotation.coordinate = CLLocationCoordinate2D()
+annotation.title = "Bobby's Coffee"
+annotation.subtitle = "Coffeeshop"
+mapView.addAnnotation(annotation)
+```
+
+See the `MGLMapViewDelegate` method `-mapView:annotationCanShowCallout:` and similar methods for allowing interaction with a callout.
+
+## Displaying annotations
+
+There are two basic ways to display the annotations you've added to a map, each with their own tradeoffs.
+
+### Annotation Images (`MGLAnnotationImage`)
+
+Annotation images are the quickest and most performant ways to display annotations, and you can provide your own custom annotation images ([example](http://mapbox.com/ios-sdk/examples)).
+
+By default, annotations added to the map are displayed with a red pin. If you'd like to customize this annotation, you can implement `MGLMapViewDelegate` `-mapView:imageForAnnotation:`.
+
+**Pros**
+
+* The easiest way to display a marker on a map
+* Easily customizable with any `UIImage`
+* High performance as the images are rendered directly in OpenGL
+
+**Cons**
+
+* Annotation images are purely static
+* No control over z-ordering
+
+### Annotation Views (`MGLAnnotationView`)
+
+If you're looking to add custom UIViews or have annotations that are dynamic or animatable, consider an `MGLAnnotationView` instead of an `MGLAnnotationImage` ([example](http://mapbox.com/ios-sdk/examples)).
+
+You'll need to implement `MGLMapViewDelegate` `-mapView:viewForAnnotation` and provide a custom `MGLAnnotationView` (`UIView`) subclass.
+
+**Pros**
+
+* Custom, native UIViews
+* Full support for animations
+* Familiar API to MapKit
+* Relative control over z-ordering using the `zPosition` property on `CALayer`
```

Comment:
Here’s a good opportunity to include a bulleted list of concrete scenarios that would be best served by annotation views (and likewise for the other two APIs). For example:

> You want to display all your users’ locations on the map with their profile pictures. Each user has a different picture.